crates/re_viewer/src/log_db_hub.rs

```diff
@@ -0,0 +1,151 @@
+use re_data_store::LogDb;
+use re_log_types::{StoreId, StoreKind};
+
+/// Stores many [`LogDb`]s of recordings and blueprints.
+#[derive(Default)]
+pub struct LogDbHub {
+    // TODO(emilk): two separate maps per [`RecordingType`].
```

Comment:
```suggestion
    // TODO(emilk): two separate maps per [`StoreKind`].
```
At that point is there much benefit to a singular StoreHub? Should there just be a RecordingHub and a BlueprintHub?

Reply:
With the current design having them together is a lot simpler when e.g. loading an .rrd, as everything in on .rrd goes into a single StoreHub